src/nunit.analyzers/Extensions/AttributeArgumentSyntaxExtensions.cs

```diff
@@ -125,13 +125,13 @@ private static string GetQualifiedTypeName(ITypeSymbol targetType)
         {
             // Note that this does not take into account generics,
             // so if that's ever added to attributes this will have to change.
-            var namespaces = new List<string>();
+            var namespaces = new Stack<string>();
```

Comment:
Not related to current PR, just thing that I mentioned. 
Namespaces go in reverse order if getting them using ContainingNamespace (e.g. "A.B.C.SomeClass" -> "C.B.A"), therefore we need to reverse them again.

Reply:
I think you are right. As far as I can remember we only hit this code for classes in the `System` namespace, so have not noticed discovered the problem.